docs/_includes/ds6/icon.html

```diff
@@ -9,12 +9,12 @@ <h3>@ebay/skin/icon/foreground</h3>
 
     <div class="demo">
         <div class="demo__inner">
-            <svg aria-hidden="true" class="icon icon--arrow-left" focusable="false" height="24" width="24">
+            <svg aria-hidden="true" class="icon icon--arrow-left" focusable="false" height="19" width="21">
                 <use xlink:href="#icon-arrow-left"></use>
             </svg>
         </div>
         {% highlight html %}
-<svg class="icon icon--arrow-left" focusable="false" height="24" width="24">
+<svg class="icon icon--arrow-left" focusable="false" height="19" width="21">
```

Comment:
Does this also need `aria-hidden="true"` for the example output?

Reply:
We really need to expand on this example and explain when to use aria-hidden vs when to use an aria-label, i.e. presentational vs non-presentational icons. If it's okay with you, I'll tackle that as part of #472 before v7 goes live.

Reply:
Sounds good.